WooCommerce/UITestsFoundation/Screens/Payments/PaymentsScreen.swift

```diff
@@ -41,6 +61,34 @@ public final class PaymentsScreen: ScreenObject {
         return self
     }
 
+    public func tapCollectPayment() throws -> Self {
+        collectPaymentButton.tap()
+        return self
+    }
+
+    public func enterPaymentAmount() throws -> Self {
+        app.enterText(text: "5")
+        nextButton.tap()
+        return self
+    }
+
+    public func takeCashPayment() throws -> Self {
+        takePaymentButton.waitForIsHittable(timeout: 10)
+        takePaymentButton.tap()
+
+        cashPaymentButton.waitForIsHittable(timeout: 10)
+        cashPaymentButton.tap()
+
```

Comment:
WDYT of extracting wait + tap to `XCTest+Extensions`:

```swift
    public func waitAndTap() {
        self.waitForIsHittable(timeout: 10)
        self.tap()
    }
```

and then having the code a bit shorter:

```suggestion
        takePaymentButton.waitAndTap()
        cashPaymentButton.waitAndTap()
```

Reply:
good idea! and this could also be reused in other cases too - updated in https://github.com/woocommerce/woocommerce-ios/pull/9621/commits/8b0485e1e7d6d582883307cf1f8e2f4547e53248